plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleConfig.java

```diff
@@ -141,6 +142,19 @@ public OracleConfig setInactiveConnectionTimeout(Duration inactiveConnectionTime
         return this;
     }
 
+    public Optional<@Min(0) Integer> getJdbcFetchSize()
+    {
+        return Optional.ofNullable(jdbcFetchSize);
+    }
+
+    @Config("oracle.jdbc-fetch-size")
```

Comment:
wondering about `jdbc-` prefix:
1. Is it needed? 
2. Can it be ommited?
3. Shoud it be `jdbc.` ?
4. Should it be something else?
5. Or it's ok as it is now?

Reply:
I'm fine with `fetch-size`